app/src/main/java/com/pajato/android/gamechat/common/PlayModeManager.java

```diff
@@ -87,17 +88,18 @@ public void handlePlayModeUserSelection(View view, BaseExperienceFragment fragme
         if (payload == null || !(payload instanceof PlayModeMenuEntry))
             return;
         PlayModeMenuEntry entry = (PlayModeMenuEntry) payload;
-        Account member = MemberManager.instance.getMember(entry.groupKey, entry.accountKey);
+        Account member = MemberManager.instance.getMember(entry.groupKeyList.get(0), entry.accountKey);
```

Comment:
In the cases where you are accessing the first element, you might want to check that that element exists to avoid an out of bounds error.